sdk/communication/azure-communication-chat/src/test/java/com/azure/communication/chat/ChatClientTestBase.java

```diff
@@ -45,9 +44,9 @@
     protected static final String CONNSTRING = Configuration.getGlobalConfiguration()
         .get("COMMUNICATION_SERVICES_CONNECTION_STRING", "pw==");
 
-    protected static final FixedDelay RETRY_STRATEGY = new FixedDelay(3, Duration.ofMillis(500));
+    protected static final FixedDelay RETRY_STRATEGY = new FixedDelay(3, Duration.ofMillis(1000));
 
-    protected ChatClientBuilder getChatClientBuilder(String token) {
+    protected ChatClientBuilder getChatClientBuilder(String token, HttpClient client) {
```

Comment:
Can we make the naming consistent, I see both client and httpClient getting used. I would prefer that everything is named httpClient so that it is easy to differentiate with the actual (sms, chat) clients

Reply:
Sounds good. Will update the names